docs/release-source/release/utils.md

```diff
@@ -36,13 +36,13 @@ Creates a new object with the given function as the protoype and stubs all imple
     class Container {
         contains(item) { /* ... */ }
     }
-    
+
     var stubContainer = sinon.createStubInstance(Container);
     stubContainer.contains.returns(false);
     stubContainer.contains.withArgs("item").returns(true);
 ```
 
-The given constructor function is not invoked. See also the [stub API](../stubs).
+The given constructor function is not invoked. See also the [stub API](./stubs.md).
```

Comment:
Hmm, strange. I was going off of the links as viewed from github rather than sinonjs.org. The link is broken if viewed from https://github.com/sinonjs/sinon/blob/master/docs/release-source/release/utils.md because `stubs.md` is in the same directory (`release/`) as `utils.md`.

Reply:
I know. But the docs are built for the homepage, not GitHub. It's just source. See `docs/CONTRIBUTING.md` for how to build and test the docs.